index.js

```diff
@@ -40,6 +40,22 @@ var decode = function(str) {
   }
 }
 
+/**
+ * Ensure any potential prototype pollution can be sanitized.
+ *
+ * @param {String} str
+ * @return {String}
+ * @api public
```

Comment:
This is not part of the public API.

```suggestion
 * @api private
```